inc/targetticket.class.php

```diff
@@ -501,6 +613,123 @@ function change_tag_type() {
          echo '</tr>';
       }
 
+      // *******************************************************************************************
+      //  Condition de génération de ticket
+      // *******************************************************************************************
+
```

Comment:
this part is interesting, because some people ask for it. Is it possible to make a PR with this feature only to allow a merge in a future major version without waiting for this whole PR be reworked ?